### DISCUSSION TITLE
meson 包构建失败，xmake 是否需要为其打补丁

### DISCUSSION THREAD

**Discussion by star-hengxing** [xmake-io/xmake#4781] [General]
这个包 https://github.com/xmake-io/xmake-repo/pull/3078 在 mingw 编译报错，具体原因在 meson 提了 https://github.com/mesonbuild/meson/issues/12913

临时解决方法是，生成 configs_file 的时候，给 windres 加上 .exe 后缀。

**Comment by waruqi** [comment]
https://github.com/xmake-io/xmake/commit/f86fca0295c477587792ec8955609966790b7dcd
